src/main/java/com/myhome/bootstrap/DataLoader.java

```diff
@@ -59,7 +50,6 @@
 
   @Override public void run(String... args) throws Exception {
     //loadPostmanData();
```

Comment:
Like @DDmit04 mentioned. We have to look for a way to include this in the `data.sql` file.

Reply:
please create new task for it